src/src/Commands/CustomTests/RunE2ECommand.php

```diff
@@ -146,6 +146,17 @@ protected function execute( InputInterface $input, OutputInterface $output ): in
 		$source              = $input->getOption( 'source' ) ?? $woo_extension;
 		$sut_action          = $input->getOption( 'sut_action' );
 
+		// Prevent usage of "--woo" and "--plugin woocommerce" together.
+		if ( ! empty( $woocommerce_version ) && ! empty( $input->getOption( 'plugin' ) ) ) {
+			foreach ( $input->getOption( 'plugin' ) as $p ) {
+				if ( $p === 'woocommerce' ) {
+					$output->writeln( '<error>Cannot use "--woo" and "--plugin woocommerce" at the same time. Use one or another.</error>' );
```

Comment:
Minor suggestion on different messaging for the error:

```suggestion
					$output->writeln( '<error>Both "--woo" and "--plugin woocommerce" cannot be used at the same time. Please use one or another when running the test.</error>' );
```